node/network/protocol/src/peer_set.rs

```diff
@@ -75,7 +75,7 @@ impl PeerSet {
 				max_notification_size,
 				set_config: SetConfig {
 					// Non-authority nodes don't need to accept incoming connections on this peer set:
-					in_peers: if is_authority == IsAuthority::Yes { 100 } else { 0 },
+					in_peers: if is_authority == IsAuthority::Yes { 25 } else { 0 },
```

Comment:
```suggestion
					in_peers: if is_authority == IsAuthority::Yes { 100 } else { 0 },
```

Reply:
For now it seems that what is in the PR currently is sufficient for improvements in block times. I would like to just roll back the changes in this one commit. Should it not be 'enough' we can revisit this change.

Reply:
This is because we only rolled out this change to a parachain with less than 20 collators.

Reply:
But aren't collator supposed to connect only when they have a block to propose ?

Reply:
Yes, this is true for 0.9.15, but we changed that in 0.9.16. The proper fix for the connections will be implemented in #4381, this is being worked on. We want to keep the slots at 100 so that we don't make it worse for 0.9.16 collators and there is not much hard done anyway.